src/styles/text-only-page.scss

```diff
@@ -11,11 +11,4 @@ a > em {
   font-weight: 600;
 }
 
-.table {
```

Comment:
Would it make sense to move the rest of this file to `tailwind.css` base layer?

Reply:
I think the `em` tag is being used incorrectly here, so perhaps better to just remove these. I'll look into it!